server/services/v1/query_runner.go

```diff
@@ -498,6 +507,48 @@ func (runner *StreamingQueryRunner) iterate(ctx context.Context, reader RowReade
 	return reader.Err()
 }
 
+// SearchQueryRunner is a runner used for Queries that are reads and needs to return result in streaming fashion
+type SearchQueryRunner struct {
+	*BaseQueryRunner
+
+	req       *api.SearchRequest
+	streaming SearchStreaming
+}
+
+func (runner *SearchQueryRunner) Run(ctx context.Context, tx transaction.Tx, tenant *metadata.Tenant) (*Response, context.Context, error) {
+	log.Debug().Msg("Search request:" + runner.req.String())
+
+	resp := api.SearchResponse{
```

Comment:
remove the dummy resp, just return `nil, ctx, api.Errorf(Code_METHOD_NOT_ALLOWED, "not implemented)` You may need to also add `Code_METHOD_NOT_ALLOWED` in errors.go.